src/modules/verifiable-credentials/types/verifiable-credentials.types.ts

```diff
@@ -28,6 +29,19 @@ export interface RoleCredentialSubjectParams {
   /** Indicates if credential is actual of time of verification */
   credentialStatus?: StatusList2021Entry;
 }
+export const validateRoleCredentialSubject = (
+  subject: RoleCredentialSubjectParams
+) => {
+  const field = subject.issuerFields?.find(
+    (field) => !['string', 'number'].includes(typeof field.value)
+  );
+  if (field) {
```

Comment:
I think this is a bit easier to read
```suggestion
  const invalidField = subject.issuerFields?.find(
    (field) => !['string', 'number'].includes(typeof field.value)
  );
  if (invalidField) {
```